client/my-sites/activity/activity-log/index.jsx

```diff
@@ -406,12 +429,24 @@ class ActivityLog extends Component {
 			};
 		} )();
 
+		const provisioningText = this.supportsRealtimeBackup()
+			? translate(
+					"We're currently backing up your site for the first time, and we'll let you know when we're finished. " +
+						"After this initial backup, we'll save future changes in real time."
+			  )
+			: translate(
+					"We're currently backing up your site for the first time, and we'll let you know when we're finished. " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'We're currently backing up your site for the first time, and we'll let you know when we're finished. After this initial backup, we'll save future changes in real time.' )` *ES Score: 9*

Reply:
This is expected, we're introducing a new variation of that string.